smac/facade/abstract_facade.py

```diff
@@ -92,8 +92,10 @@ class AbstractFacade:
         Callbacks, which are incorporated into the optimization loop.
     overwrite: bool, defaults to False
         When True, overwrites the run results if a previous run is found that is
-        inconsistent in the meta data with the current setup. If ``overwrite`` is set to False, the user is asked
-        for the exact behaviour (overwrite completely, save old run, or use old results).
+        consistent in the meta data with the current setup. When False and a previous run is found that is
+        consistent in the meta data, the run is continued. When False and a previous run is found that is
```

Comment:
FYI: Not sure if I misunderstand something, but I would say it needs to be "consistent" instead of "inconsistent" here.

Reply:
i think you are right

Reply:
+1